gui/default/index.html

```diff
@@ -329,9 +332,39 @@ <h3 class="panel-title">
         </div>
       </div>
 
-      <!-- First regular row -->
+      <!-- Login form -->
+      <div ng-if="!authenticated" class="center-block">
+        <h3 translate>Authentication required</h3>
```

Comment:
```suggestion
        <h3 translate>Authentication Required</h3>
```

Uppercase to make it consistent with other `h3`s (e.g. `This Device`, `Remote Devices`, etc.).